packages/handshake-api/src/index.js

```diff
@@ -36,18 +37,36 @@ const getDomainRecords = async (name) => {
   return records;
 };
 
-const findSkylinkRecord = (records) => {
+const findSkynetCompatibleRecord = (records) => {
   // Find the last one, so people can update their domains in a non-destructive
   // way by simply adding a new link. This will also allow keeping links to
   // older versions for backwards compatibility.
   return records
     ?.slice()
     .reverse()
-    .find(({ txt }) => txt?.some((entry) => isValidSkylink(entry)));
+    .find(({ txt }) => txt?.some((entry) => isValidSkylink(entry) || isValidRegistryEntry(entry)));
 };
 
-const getSkylinkFromRecord = (record) => {
-  return record?.txt?.find((entry) => isValidSkylink(entry));
+const createResponseFromCompatibleRecord = (record) => {
+  const skylink = record?.txt?.find((entry) => isValidSkylink(entry));
+
+  if (skylink) return { skylink };
```

Comment:
In the nginx config we prioritize registry entries over skylinks and I think that's a good idea. Here we do the opposite. I believe we should be prioritizing registry entries but more than that we should be consistent.

Reply:
We don't prioritise registry entries over skylinks -  the prority lies in an order of domain configuration entries. We find latest domain configuration entry that matches either a skylink or skydb entry in `findSkynetCompatibleRecord` and it can only contain either a skylink or a skydb entry so the order of if-else in the code doesn't really matter (and actually in both nginx and api, we first check for skylink existence, then we check for registry entry existence).

Reply:
OK.